packages/eval/harbor/test_relay_contract.py

```
@@ -446,6 +461,195 @@ async def test_check_is_scoped_to_runs_that_require_the_proxy(self):
+        #   (command, cwd=None, env=None, timeout_sec=None, user=None)
+        # A **kwargs-only double would absorb user= and hide a TypeError
+        # against that signature.
+        parameter = inspect.signature(SubjectEnvironment.exec).parameters["user"]
+        self.assertNotEqual(parameter.kind, inspect.Parameter.VAR_KEYWORD)
+        inspect.signature(SubjectEnvironment.exec).bind(
+            None, "true", user="root"
+        )
+
+    def test_published_ipv4_requires_four_0_255_octets(self):
+        relay = load_relay()
+        accepted = ("172.18.0.2", "0.0.0.0", "255.255.255.255", "10.0.0.1")
```

Ruff (0.16.1)

[error] Possible binding to all interfaces

(S104)